From f4bac3c5247bfda27bff1b376723289e3a912fbb Mon Sep 17 00:00:00 2001
From: Sara Golemon <sgolemon@fb.com>
Date: Wed, 14 Jan 2015 10:14:18 -0800
Subject: [PATCH] Install hphp/runtime/version.h during make install

Summary: Needed for DSO compiles

Closes #4639

Reviewed By: @JoelMarcey, @fredemmott

Differential Revision: D1781936
---
 hphp/runtime/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hphp/runtime/CMakeLists.txt b/hphp/runtime/CMakeLists.txt
index 9816330..f328ae1 100644
--- a/hphp/runtime/CMakeLists.txt
+++ b/hphp/runtime/CMakeLists.txt
@@ -2,6 +2,7 @@ set(RECURSIVE_SOURCE_SUBDIRS base debugger eval server vm)
 foreach (dir ${RECURSIVE_SOURCE_SUBDIRS})
   HHVM_SELECT_SOURCES("${CMAKE_CURRENT_SOURCE_DIR}/${dir}")
 endforeach(dir ${RECURSIVE_SOURCE_SUBDIRS})
+list(APPEND HEADER_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/version.h")
 
 if(NOT LINUX)
   add_definitions(-DNO_HARDWARE_COUNTERS)
